sound/soc/codecs/rt700.c

```diff
@@ -1133,6 +1133,11 @@ int rt700_io_init(struct device *dev, struct sdw_slave *slave)
 	if (rt700->hw_init)
 		return 0;
 
+	if (rt700->first_init) {
+		regcache_cache_only(rt700->regmap, false);
+		regcache_cache_bypass(rt700->regmap, true);
```

Comment:
and first_init is really 'first_boot'. This is something we done once, and the variable is not reset for suspend-resume, so the commit message is confusing to me.